.github/workflows/ext-build-subtopology-ac.yml

```diff
@@ -0,0 +1,87 @@
+# Runs the subtopology autocoder tool to test the subtopology tool (https://github.com/mosa11aei/fprime-subtopology-tool)
+
+name: "Subtopology Autocoder Tool Test"
+
+on:
+  push:
+    branches: [ devel, release/** ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ devel, release/** ]
+    paths-ignore:
+      - 'docs/**'
+      - '**.md'
+      - '.github/actions/spelling/**'
+      - '.github/ISSUE_TEMPLATE/**'
+
+jobs:
+  get-branch:
+    name: "Get target branch"
+    uses: ./.github/workflows/reusable-get-pr-branch.yml
+    with:
+      target_repository: mosa11aei/fprime-rngLibrary
+
+  subtopology-test:
+    name: "Subtopology Test"
+    runs-on: ubuntu-latest
+    needs: get-branch
+    steps:
+      - name: "Checkout target repository"
+        uses: actions/checkout@v4
+        with:
+          submodules: false
+          repository: mosa11aei/fprime-rngLibrary
+          ref: ${{ needs.get-branch.outputs.target-branch }}
+      - name: "Overlay current F´ revision"
+        uses: actions/checkout@v4
+        with:
+          submodules: true
+          path: ./fprime
+          fetch-depth: 0
+      - uses: ./fprime/.github/actions/setup
+        with:
+          location: ./fprime
+      - name: "Verify CMake is installed"
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y cmake
+      - name: "Get FPP"
+        uses: actions/checkout@v4
+        with:
+          submodules: false
+          repository: nasa/fpp
+          path: ./fpp
+          ref: main
+      - name: "[fpp] Install sbt"
```

Comment:
Get rid of non native fpp install

Reply:
@mosa11aei if it were me I would get rid of all the installs, since all those tools come pre-installed on the GitHub runners. This may prevent breakages in the future. Your call.

Reply:
This specifically relates to something @LeStarch  and I talked about today. See https://github.com/nasa/fprime/pull/2816 😉